packages/business-rules-lib/src/validators/contact.js

```diff
@@ -286,3 +286,25 @@ export const createLastNameValidator = joi =>
     .external(capitaliseNamePrefixes(['Mc', "O'"]))
     .required()
     .example('Tester')
+
+/**
+ * Create a validator to check a valid national insurance number
+ *
+ * @param {Joi.Root} joi the joi validator used by the consuming project
+ * @returns {Joi.AnySchema}
+ */
+
+const ukNINORegEx = /^([ABCEGHJ-PRSTW-Z][ABCEGHJ-NPRSTW-Z])\s?([0-9]{2})\s{0,3}([0-9]{2})\s{0,3}([0-9]{2})\s{0,3}([ABCD])$/
```

Comment:
I'm not 100% this is correct. My NI number starts 'JL', but looks like the L isn't a valid second character

Reply:
The expression does allow L as this is part of the J-N range defined as allowed for the second character